python/ray/serve/autoscaling_policy.py

```diff
@@ -129,6 +55,7 @@ def get_decision_num_replicas(
         return curr_target_num_replicas
 
 
+@PublicAPI(stability="stable")
```

Comment:
"Basic" is quite a bad name. It's unclear why it's "basic" and it doesn't describe how the policy works at all...

Let's rename it. I'm open to something like:

- `DefaultAutoscalingPolicy` (makes it clear that it's the default but doesn't tell you what it does)
- `ReplicaQueueLengthAutoscalingPolicy` (describes the behavior but doesn't tell you it's the default)

Or maybe we could have `DefaultAutoscalingPolicy` as an alias for `ReplicaQueueLengthAutoscalingPolicy`